components/organisms/footer/index.tsx

```diff
@@ -1,7 +1,63 @@
-import { FooterStyled } from './styled';
+import { 
+  FooterStyled, 
+  ContentStyled, 
+  AsideStyled, 
+  ParagraphStyled, 
+  TitleStyled, 
+  ContactStyled, 
+  SubtitleStyled, 
+  ContactInfoStyled, 
+  CopyrightStyled,
+  InfoStyled,
+  PhraseStyled,
+} from './styled';
+import Image from 'next/image';
+import logo from '../../../assets/images/Logo-Small.svg';
+import heart from '../../../assets/images/Vector.svg';
 
 const Footer = () => {
-  return <FooterStyled>Footer</FooterStyled>;
+  return (
+    <FooterStyled>
+      <ContentStyled>
+        <AsideStyled>
+          <ParagraphStyled>&lt;AND DANCE&gt;</ParagraphStyled>
+        </AsideStyled>
+
+        <TitleStyled>VAMOS CONVERSAR</TitleStyled>
+
+        <ContactStyled>
+          <SubtitleStyled>nosso whatsapp</SubtitleStyled>
+          <ContactInfoStyled>+55 51 99693.9336</ContactInfoStyled>
+        </ContactStyled>
+
+        <ContactStyled>
+          <SubtitleStyled>nosso e-mail</SubtitleStyled>
+          <ContactInfoStyled>contato@l2code.com.br</ContactInfoStyled>
+        </ContactStyled>
+
+        <ContactStyled>
+          <SubtitleStyled>entre no time</SubtitleStyled>
+          <ContactInfoStyled>talentos@l2code.com.br</ContactInfoStyled>
+        </ContactStyled>
+      </ContentStyled>
+      
+      <CopyrightStyled>
+        <InfoStyled>
+          <PhraseStyled>Abraçamos os talentos do mundo</PhraseStyled>
+          <PhraseStyled>Somos uma empresa que trabalha 100% remoto</PhraseStyled>
+        </InfoStyled>
+
+        <InfoStyled>
+          <Image src={logo} alt="L2 Code" />
```

Comment:
Aqui na volta dessa imagem podemos colocar um link com "/" para que toda vez que for clicado, o usuário seja direcionado para o home.